docs/HowToGuides/FirstPullRequest.md

```diff
@@ -169,11 +167,13 @@ something in the near future. Please don't feel sad or apologetic!
 
 ## I submitted and merged my first pull request. What now?
 
-Awesome! You could try fixing a few more Starter Bugs until you feel some
-level of comfort working with the codebase. You could also start looking at
-other bugs which interest you and you think you might be able to tackle.
-Don't forget to ask for help if you need directions or you get stuck!
+Awesome! You could try fixing a few more [good first issues][good-first-issues]
+until you feel some level of comfort working with the codebase. You could also
+start looking at other bugs which interest you and you think you might be able
```

Comment:
Could you please slightly re-phrase this sentence to drop some of the "you"?

Reply:
I just noticed this 🤣